modules/theme/src/themes/default/collections/table.overrides

```diff
@@ -27,6 +27,13 @@
     }
 }
 
+.ui.table {
```

Comment:
Is this style reusable in other portals.
IMO, no. And this should come in `developer-portal.less`(Style sheet meant for Console).

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3566/commits/66c501e3bafdfeba484d242847e688d8d0ee2611

Reply:
Lets move `.thead.user-attribute-table-header {` as well.